switchbot_ros/scripts/switchbot_ros_server.py

```diff
@@ -97,7 +162,7 @@ def execute_cb(self, goal):
             command_type = 'command'
         try:
             if not self.bots:
-                self.bots = SwitchBotAPIClient(token=self.token)
+                self.bots = SwitchBotAPIClient(token=self.token,secret=self.secret)
```

Comment:
```suggestion
                self.bots = SwitchBotAPIClient(token=self.token, secret=self.secret)
```

Reply:
( I wrote this comment at another unrelated conversation. Sorry. )
A Space added as you suggested.
https://github.com/jsk-ros-pkg/jsk_3rdparty/commit/429cd88063ae3b2785f682388c4939607b507d73#diff-1690cbebb7bf92ad5c7be4524a54f4c0566916a4da9a6b0c493666e891db80c9R165